amazonclouddrive/amazonclouddrive.go

```diff
@@ -597,6 +597,110 @@ func (f *Fs) Mkdir() error {
 	return f.dirCache.FindRoot(true)
 }
 
+// Move src to this remote using server side move operations.
+//
+// This is stored with the remote path given
+//
+// It returns the destination Object and a possible error
+//
+// Will only be called if src.Fs().Name() == f.Name()
+//
+// If it isn't possible then return fs.ErrorCantMove
+func (f *Fs) Move(src fs.Object, remote string) (fs.Object, error) {
+	srcObj, ok := src.(*Object)
+	if !ok {
+		fs.Debug(src, "Can't move - not same remote type")
+		return nil, fs.ErrorCantMove
+	}
+	fs.Debug(src, "\n\nTrying to move '%s' -> '%s'", srcObj.Remote(), remote)
```

Comment:
I'd write this as a log on the object like this.  As you passed `src` as the first parameter you'll get `srcObj.Remote()` at the start of the log anyway.

    fs.Debug(srcj, "Attempting to move to %q'", remote)